nio_cli/commands/config.py

```diff
@@ -26,8 +26,58 @@ def config_project(name='.'):
                 tmp.write('PK_TOKEN={}\n'.format(pk_token))
             else:
                 tmp.write(line)
-        os.remove(env_location)
-        os.rename(tmp.name, env_location)
+        os.remove(conf_location)
+        os.rename(tmp.name, conf_location)
+    secure = input('Optional secure instance configuration [Y/N]: ')
+    if secure.lower() == 'y':
+        config_ssl(name, conf_location)
+
+# Assumes the user has OpenSSL already installed
+
+
+def config_ssl(name, conf_location):
+    ssl_cert = ''
+    ssl_key = ''
+
+    new_certs = input('Generate a self-signed certificate/key [Y/N]: ')
+
+    if (new_certs.lower() == 'y'):
+        subprocess.call('mkdir ssl &&  cd ssl')
```

Comment:
Looks like `subprocess` needs to be imported

Reply:
Also, add a `shell=True` to the end of this guy. Otherwise I get ` No such file or directory: 'mkdir ssl && cd ssl'`